akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala

```diff
@@ -219,8 +219,8 @@ object ActorSystem {
     apply(guardianBehavior, name, config)
 
   /**
-    * Java API: Create an ActorSystem
-    */
+   * Java API: Create an ActorSystem
+   */
```

Comment:
if you are using "format on save" in IntelliJ you have to approve download of new version of scalafmt. It's a small notice in status bar (event log)